Development-Guide/CAN-Filter-Guide/README.adoc

```diff
@@ -2,44 +2,44 @@
+There are two components to a CAN filter: an ID and a Mask.
+The ID is the unique identifier for the address of the message(s) being looked for, and the Mask determines which bits of the ID are important to sift through. 
 
-IMPORTANT: A Mask must be able to adapt to future additions to a CAN network. It must be made *as tight as possible for longevity*.
+IMPORTANT: A Mask must be able to adapt to future additions to a CAN network. 
+It must be made *as tight as possible for longevity*.
 
-There is no set order to follow the steps, but to get started follow the steps below until you are comfortable.
-Another tip is to convert the hex format into binary format, determine the ID and Mask, and then convert those back into hex.
+There is no set order of steps to follow, but to get started, determine the ID first, then determine the Mask.
+A very useful tip is to convert the hex format into binary format, determine the ID and Mask, and then convert those back into hex.
 
-== Steps
 
-. Determine the ID
+== Determine the ID
 
-.. The ID will be the unique identifier for the message(s) that you are trying to receive
+. The ID will be the unique identifier for the address of the message(s) being received.
 
-.. The ID should be very specific and contain the important bits that will be compared to the incoming message.
+. The ID should be very specific and contain the important bits that will be compared to the incoming message.
 
-The greater the variery of addresses, the more flexible your ID will be to accommodate the variety of incoming messages.
+The greater the variety of addresses wanted, the more flexible the ID will be to accommodate the variety of incoming messages.
 
-.. For example, If you are looking for a message from a system that sends out messages with address 0b0000_1111(0x0F) then you want your ID to look like 0b0000_1111(0x0F) because there is only one type of address you want to receive.
+. For example, if one is looking for a message from a system that sends out messages with address 0b0000_1111(0x0F) then the ID will be 0b0000_1111(0x0F) because there is only one type of address that is wanted.
 
-.. For example, If you are looking for a message from a system that sends out messages with addresses 0b0001_0001, 0b0001_0010, and 0b0001_0011 (0x11, 0x12, and 0x13) then you want your ID to look like 0b0001_0000 (0x10) because the last two bits do not matter. 
-You could choose 0x11 or 0x15 for your ID or any number for the last hex digit, and it will work regardless. 
-It is your mask that determines which bits are important to compare.
+. For example, if one is looking for a message from a system that sends out messages with addresses 0b0001_0001, 0b0001_0010, and 0b0001_0011 (0x11, 0x12, and 0x13) then the ID will be 0b0001_0000 (0x10) because the last two bits do not matter. 
```

Comment:
This doesn't really deserve it's own number since it's an example not a step in itself. Perhaps this whole section fits better as a paragraph.